tasks/managed/rh-sign-image/README.md

```diff
@@ -18,7 +18,11 @@ Task to create internalrequests or pipelineruns to sign snapshot components
 | pyxisServer              | The server type to use. Options are 'production','production-internal,'stage-internal' and 'stage'.                                                                                                                                               | Yes      | production    |
 | pyxisSecret              | The kubernetes secret to use to authenticate to Pyxis. It needs to contain two keys: key and cert                                                                                                                                                 | No       | -             |
 | signRegistryAccessPath   | The relative path in the workspace to a text file that contains a list of repositories that needs registry.access.redhat.com image references to be signed (i.e. requires_terms=true), one repository string per line, e.g. "rhtas/cosign-rhel9". | No       | -             |
+| batchLimit              | size of batch attributes to send to internal-request. As internal request arguments are need to be strings, size here represent maximal string length of `references` and `manifest_digests` sent to internal request                             | Yes      | 4096          |
 
+## Changes in 5.0.4
```

Comment:
This is quite a significant change and a new feature, not just a bugfix, so should be 5.1.0. There should probably also be a note saying that the utils image is bumped and as a result, `--task-git-*` params are no longer passed for the `internal-*` script.

Reply:
>>  There should probably also be a note saying that the utils image is bumped and as a result, --task-git-* params are no longer passed for the internal-* script.
Yeah, with that one I wasn't sure if it's needed or if there's something wrong on my side. I'll put it into the readme as well.